STX.REST.RESTFulSense.Clients/Models/ErrorMappers/Exceptions/ErrorMapperValidationException.cs

```diff
@@ -0,0 +1,15 @@
+// ----------------------------------------------------------------------------------
+// Copyright (c) The Standard Organization: A coalition of the Good-Hearted Engineers
+// ----------------------------------------------------------------------------------
+
+using Xeptions;
+
+namespace STX.REST.RESTFulSense.Clients.Models.ErrorMappers.Exceptions
+{
+    public class ErrorMapperValidationException : Xeption
+    {
+        public ErrorMapperValidationException(Xeption innerException)
+            : base(message: "Error mapper validation errors occured, please try again.", innerException)
+        { }
```

Comment:
You need to add your message in the service.  There was an upgrade to the standard a while back.

```suggestion
        public ErrorMapperValidationException(string message, Xeption innerException)
            : base(message, innerException)
        { }
```

Reply:
Treated